src/components/Tag/Tag.tsx

```diff
@@ -0,0 +1,83 @@
+import clsx from 'clsx';
+import React from 'react';
+import styles from './Tag.module.css';
+import Text from '../Text';
+
+export type Color =
+  | 'neutral'
+  | 'alert'
+  | 'success'
+  | 'warning'
+  | 'revise'
```

Comment:
We should make a note to make the code names and the design names for these things the same – in figma they're called "default", "stop", "go", "warning", "yield", and "brand" (only "warning" and "brand" are the same).

https://www.figma.com/file/P9VPyI821gLq832tuU2WeY/UI-Kit-Master-(Core)?node-id=2525%3A662
<img width="757" alt="the tag component in figma" src="https://user-images.githubusercontent.com/7761701/161808694-ecde3374-3fb7-4fc5-b523-ddced7cd7ad1.png">

Reply:
Should I change these to "default", "stop", "go", "warning", "yield", and "brand"?

Reply:
Will change to "default", "stop", "go", "warning", "yield", and "brand"

Reply:
We're still going to need to normalize these with design because they're using the other pattern for `Banner` 😭 I'll make a ticket.
<img width="1014" alt="Banner component in figma" src="https://user-images.githubusercontent.com/7761701/161836945-5e315de7-9247-4d73-861f-88585c216eef.png">

Reply:
Sweet, thanks

Reply:
Ticket: https://app.shortcut.com/czi-edu/story/191591/make-banner-and-tag-variant-names-consistent